### PR TITLE
Add test coverage to opensrp-register module enhancement

### PR DESCRIPTION
**Pull request by proshantokuet**

### CODE REVIEW

File: opensrp-register/src/main/java/org/opensrp/register/service/handler/ANCScheduleHandler.java [hunk 1]
```diff
@@ -9,10 +9,13 @@
 import org.springframework.stereotype.Component;
 
 @Component
-public class ANCScheduleHandler extends BaseScheduleHandler {
+public class ANCScheduleHandler extends BaseScheduleHandler {
```
Comment (pld) [inline opensrp-register/src/main/java/org/opensrp/register/service/handler/ANCScheduleHandler.java]: why extra spaces here? and below should have a space between `)` and `{`

File: opensrp-register/src/main/java/org/opensrp/register/service/handler/BNFScheduleHandler.java [hunk 2]
```diff
@@ -11,28 +11,31 @@
...
+        this.bnfSchedulesService = bnfSchedulesService;
+    }
+    @Override
+    public void handle(Event event, JSONObject scheduleConfigEvent,String scheduleName) {
+        try {
 			
-			if(scheduleName==null){
-				scheduleName="BirthNotificationPregnancyStatusFollowUp";
-			}
+            if(scheduleName==null){
```
Comment (pld) [inline opensrp-register/src/main/java/org/opensrp/register/service/handler/BNFScheduleHandler.java]: here and elsewhere in this file missing spaces for formatting

**Comment by pld** [comment]
@proshantokuet all those code-indentation commits should be a single commit, I don't see a compelling reason to split that across multiple commits, and it's definitely a bad sign if multiple commits have the same commit message (that often indicates they'd be better as a single commit). I'd suggest you use an interactive rebase to combine those commits into a single "code indentation" commit, https://github.com/onaio/ona-tech/blob/master/github.md#rebasing

File: opensrp-register/src/main/java/org/opensrp/register/service/handler/ANCScheduleHandler.java (hunk 1 above)
Comment (proshantokuet) [inline opensrp-register/src/main/java/org/opensrp/register/service/handler/ANCScheduleHandler.java]: please check.

File: opensrp-register/src/main/java/org/opensrp/register/service/handler/BNFScheduleHandler.java (hunk 2 above)
Comment (proshantokuet) [inline opensrp-register/src/main/java/org/opensrp/register/service/handler/BNFScheduleHandler.java]: Please check.

**Pull request merged**